SeeShellsV2/SeeShellsV2/Data/RegistryShellbagRoot.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.ComponentModel;
+using System.Collections.ObjectModel;
+
+namespace SeeShellsV2.Data
+{
+    /// <summary>
+    /// The root of a shellbag tree from a registry hive
+    /// </summary>
+    public class RegistryShellbagRoot : INotifyPropertyChanged
+    {
+        /// <summary>
+        /// Name of the source registry hive
+        /// </summary>
+        public string Name { get; init; }
+
+        /// <summary>
+        /// absolute path to the registry hive
+        /// </summary>
+        public string Path { get; init; }
+
+        /// <summary>
+        /// Top level children of the shellbag root
+        /// </summary>
+        public ObservableCollection<IShellItem> Children { get; init; }
+
+        public event PropertyChangedEventHandler PropertyChanged;
```

Comment:
PropertyChanged is never used. Is it going to be used in the future? Or is it fine how it is?

Reply:
I think it is used by code generated by WPF. I am not an expert on this, but my understanding is that WPF views will not be able to react to changes to the object (bound to the registry view) unless that is present. I will run a quick test to see if it is necessary.

Reply:
Ok after a quick test it seems that it is not necessary because the `Name` and `Path` properties are read-only. I will prepare a commit to remove it (along with any other changes you would like to see).

Reply:
Everything else looks good. It just had a warning on the use of propertychanged so I figured I'd ask. Once that's committed then I'll approve.